src/istio/control/http/request_handler_impl_test.cc

```diff
@@ -403,8 +403,9 @@ TEST_F(RequestHandlerImplTest, TestDefaultApiKey) {
                           TransportCheckFunc transport,
                           CheckDoneFunc on_done) -> CancelFunc {
         auto map = attributes.attributes();
-        EXPECT_EQ(map[AttributeName::kRequestApiKey].string_value(),
-                  "test-api-key");
+        EXPECT_EQ(
+            map[istio::utils::AttributeName::kRequestApiKey].string_value(),
```

Comment:
nit:  if a module is under istio namaespace,  you can remove namespace prefix "istio::"

Reply:
Done.